osu.Game/Overlays/Profile/Header/Components/OverlinedInfoContainer.cs

```diff
@@ -12,6 +12,11 @@ namespace osu.Game.Overlays.Profile.Header.Components
 {
     public class OverlinedInfoContainer : CompositeDrawable
     {
+        /// <summary>
+        /// The amount of space between the overline and the underneath text.
+        /// </summary>
+        private const float line_bottom_margin = 2;
```

Comment:
No reason to introduce single-use constants. Using `2` directly where needed would also save on the xmldoc.

Reply:
honestly I just found another class that used a margin this way, but you're right, it's a bit overboard for a single use, will change